src/compiler/diagnosticMessages.json

```diff
@@ -5653,5 +5653,9 @@
     "An optional chain cannot contain private identifiers.": {
         "category": "Error",
         "code": 18030
+    },
+    "The type 'never' was reduced from the intersection '{0}'. Each type of that intersection has properties that conflict, so values of that type can never exist.": {

```

Comment:
why 'each' here? Wouldn't 'some' be correct?

If that's true, I'd just say "Types in that intersection ..."

Reply:
Good point